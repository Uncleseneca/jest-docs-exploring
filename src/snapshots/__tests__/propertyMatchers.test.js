test.skip('will fail every time', () => {
  const user = {
    name: 'Mike Vazoffski',
    age: Math.floor(Math.random() * 30),
    id: Math.floor(Math.random() * 40),
  };

  expect(user).toMatchSnapshot();
});

test('wont fail', () => {
  const user = {
    name: 'Mike Vazoffski',
    age: Math.floor(Math.random() * 30),
    id: Math.floor(Math.random() * 40),
  };

  expect(user).toMatchSnapshot({
    age: expect.any(Number),
    id: expect.any(Number),
  });
});
