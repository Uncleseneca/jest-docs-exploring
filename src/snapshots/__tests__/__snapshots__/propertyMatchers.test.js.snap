// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will fail every time 1`] = `
Object {
  "age": 2,
  "id": 3,
  "name": "Mike Vazoffski",
}
`;

exports[`wont fail 1`] = `
Object {
  "age": Any<Number>,
  "id": Any<Number>,
  "name": "Mike Vazoffski",
}
`;
